rpi_backlight/cli.py

```diff
@@ -48,6 +53,12 @@ def _create_argument_parser():
         action="version",
         version="%(prog)s {version}".format(version=__version__),
     )
+    parser.add_argument(
+        "-B",
+        "--board",
+        default=1,
+        help="Please provide a board model.\n" "0: Raspberry Pi\n" "1: TinkerBoard\n.",
+    )
```

Comment:
Let's change this to a string identifier to be closer to the Python API:

```suggestion
    board_types = {
        "raspberry_pi": BoardType.RASPBERRY_PI,
        "tinker_board": BoardType.TINKER_BOARD,
    }
    parser.add_argument(
        "-B",
        "--board-type",
        default="raspberry_pi",
        choices=board_types.keys()
        help="board type",
    )
```

We also don't have to do the validation ourselves (below) but can use `choices` instead: https://docs.python.org/3/library/argparse.html#choices